### ISSUE TITLE
Readme.md: "Download on Flathub" badge is broken

### ISSUE THREAD

**Issue by sebastian-philipp** [eneshecan/whatsapp-for-linux#416]
The Badge links to https://flathub.org/apps/com.github.xeco23.WasIstLos which is a 404. Unfortunately, I'm not really sure if this app appears under a different name there: https://flathub.org/apps/search?q=whatsapp

**Comment by xeco23** [comment]
Yeah, we are in a progress of changing the app name. There is no flathub package atm until someone takes care of it.

**Comment by euxaristia** [comment]
I just got confused by the README, perhaps the flatpak "sticker" should be omitted until resolved
